pages/wiki/synchronization-clients.md

```diff
@@ -24,6 +24,8 @@ layout: documentation
 
 <p>There is currently no application available for iOS phones to synchronize with AsteroidOS watches. As far as we know, there hasn't been any development done on such an app yet.</p>
 
+<p>One can however use generic BLE scanners/communication apps like <a href="https://apps.apple.com/gb/app/nrf-connect-for-mobile/id1054362403">nRF Connect</a> to still connect the watch to an iOS device and even use it to send notifications to the watch. Specific functionality like syncing time or weather forecast are however not supported!</p>
```

Comment:
Like you, I find that nRF Connect works better, but one can pair the watch with just the Apple settings.  Given that, I'd suggest that we might change it to say something like:

"However, if the watch is paired with an iOS phone, AsteroidOS is capable of displaying notifications from the phone.  Pairing can be done using Apples Bluetooth settings, but several iOS users have reported that a generic BLE scanner/communication app like {nRF connect with hyperlink} seems to be more reliable for pairing Bluetooth devices.  Specific functionality ...."

Reply:
Done in 68e82e7433c19172eff08d67610e3020eaa537b2

Reply:
Based om what _DocGalaxyBlock_ said in the Matrix channel ([archive link](https://log.asteroidos.org/?date=2023-02-24&level=16#184)), it seems that being able to connect an iDevice to a watch running AsteroidOS via standard Bluetooth settings seems to be the exception, rather than the norm.

They tested `sturgeon`, `sparrow` and `narwhal` on a MacBook.

![image](https://user-images.githubusercontent.com/40122905/221362926-1c21d398-0c5f-4c28-b944-c965c3a59629.png)

Reply:
That's a separate issue.  This paragraph talks about iOS, not OSX.